proto/buf.yaml

```diff
@@ -12,9 +12,27 @@ lint:
     # "legacy" lib protos.
     - teleport/lib/multiplexer/test/ping.proto
     - teleport/lib/web/envelope.proto
-    # TODO: Take care of these.
-    - teleport/lib/teleterm
-    - teleport/lib/prehog
+  ignore_only:
+    # RPC_RESPONSE_STANDARD_NAME and RPC_REQUEST_RESPONSE_UNIQUE make it impossible to follow
+    # Google's standard methods. https://cloud.google.com/apis/design/standard_methods
+    RPC_RESPONSE_STANDARD_NAME:
+      - teleport/lib/teleterm
+    RPC_REQUEST_RESPONSE_UNIQUE:
+      - teleport/lib/teleterm
+    PACKAGE_DIRECTORY_MATCH:
+      # Renaming the proto package name at this point would be a breaking change.
+      - teleport/lib/prehog
+    # TODO(ravicious): Fix protos and re-enable lib/teleterm lint rules below.
+    # TODO(espadolini): Fix protos and re-enable lib/prehog lint rules below.
+    RPC_REQUEST_STANDARD_NAME:
```

Comment:
Same as above.

Reply:
`api/proto/buf.yaml` has exceptions for

* `RPC_REQUEST_RESPONSE_UNIQUE`
* `RPC_RESPONSE_STANDARD_NAME`

Do you indeed mean that I should add `RPC_REQUEST_STANDARD_NAME` there as well?

I'm asking only because originally you added `RPC_REQUEST_STANDARD_NAME` as an exception for lib/teleterm with a comment to fix it down the line:

https://github.com/gravitational/teleport/blob/0fbec707503d04e75f9ad2b544b5c588875b4559/lib/teleterm/api/proto/buf.yaml#L12-L17

Reply:
Sorry, you are correct, we do obey `RPC_REQUEST_STANDARD_NAME`. Let's keep the exception only for Teleterm.